ui/components/ui/textarea/index.scss

```diff
@@ -0,0 +1,25 @@
+@use "design-system";
+
+.textarea {
+  display: block;
+  box-shadow: none;
+  color: design-system.$black;
+
+  @include design-system.H6;
+
+  font-size: 1rem;
```

Comment:
The mixin should take care of `font-size` `design-system.H6 = 14px` `1rem = 16px`
```suggestion
  @include design-system.H6;
```

It's 14px in the design.
<img width="753" alt="Screen Shot 2021-11-24 at 3 16 24 PM" src="https://user-images.githubusercontent.com/8112138/143324466-0f0415a3-2528-4a29-8aba-690f4982d106.png">

Reply:
Yup, I left that there on purpose. I checked with @holantonela yesterday and she said font should be 16px